.github/gi

```diff
@@ -0,0 +1 @@
+* @edigaryev @fkorotkov
```

Comment:
Shouldn't we name this file `CODEOWNERS` instead, as per [GitHub's documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location)?